src/main/scala/fm/html/Html5ConciseStringBuilder.scala

```diff
@@ -0,0 +1,19 @@
+package fm.html
+
+import java.lang.StringBuilder
+
+object Html5ConciseStringBuilder {
+  def capture(f: Html5RenderCtx => Unit): String = {
+    val builder: Html5ConciseStringBuilder = new Html5ConciseStringBuilder
+    f(builder.html5RenderCtx)
+    builder.result()
+  }
+}
+
+final class Html5ConciseStringBuilder extends Html5 {
+  private val sb: StringBuilder = new StringBuilder()
+  implicit val html5RenderCtx: Html5RenderCtx = new ConciseHtml5RenderCtx(sb)
+
+  def result(): String = sb.toString()
+  def clear(): Unit = sb.setLength(0)
+}
```

Comment:
Is there a good way to merge this with the existing Html5StringBuilder since this is basically copy/pasted code?